### PR TITLE
-Add Microchip SEC1210 UART Support.

### PR DESCRIPTION
**Pull request by FabioAraujo1977**
- Adding support for Microchip SEC1210 Dual reader.

### CODE REVIEW

**Comment by LudovicRousseau** [comment]
Thanks for the PR.
I did not know that the SEC1210 was also available with a serial connection.

File: src/reader.conf.in [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #     GemCoreSIMPro2
 #     GemPCPinPad
 #     GemPCTwin (default value)
+#     SEC1210 (Dual Uart Reader)
```
Comment (LudovicRousseau) [inline src/reader.conf.in]: I guess you mean "dual slot" here?
The way the two slots are internally managed is not really relevant for users.

File: src/ccid_serial.c [hunk 2]
```diff
@@ -833,6 +850,11 @@ status_t OpenSerialByName(unsigned int reader_index, char *dev_name)
 		unsigned char tx_buffer[] = { 0x02 };
 		unsigned char rx_buffer[50];
 		unsigned int rx_length = sizeof(rx_buffer);
+		
+		if (0 == strcasecmp(reader_name,"SEC1210"))
+		{
+			tx_buffer[0] = 0x06;
+		}
```
Comment (LudovicRousseau) [inline src/ccid_serial.c]: I think I will change the code to use a if () {} else {} here.

File: src/reader.conf.in (hunk 1 above)
Comment (FabioAraujo1977) [inline src/reader.conf.in]: Yes, you are right

File: src/ccid_serial.c (hunk 2 above)
Comment (LudovicRousseau) [inline src/ccid_serial.c]: What is this command 0x06?
The command 0x02 used to get the firmware does not work?

Comment (FabioAraujo1977) [inline src/ccid_serial.c]: when we sent 0x02 to get firmware version, 
"03 06 6B 01 00 00 00 00 00 00 00 00 02 6D"
we received
frame: "83 00 00 00 00 00 00 41 00 00" - "Command not supported."

when we sent 0x06 to get firmware version, 
"03 06 6B 01 00 00 00 00 00 00 00 00 06 69"
we received
frame: "83 01 00 00 00 00 00 01 00 00 22", "Everything OK."

**Comment by LudovicRousseau** [comment]
Committed in https://github.com/LudovicRousseau/CCID/commit/ec652d249dc81c647bf80d5d3c21665ef00c6d8b
Thanks

**Pull request closed without merging**